sdk/storage/Azure.Storage.Blobs/CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Fixed \[BUG\] Fixed BlobContainerClient constructor to set clientSideEncryptionOptions #44623
```

Comment:
Specify the constructor that got changed since the bug didn't affect all constructors.

If I remember off the top of my head it was like `BlobContainerClient(connectionstring, credential, options)` ?